components/TransactionItem.js

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+
+
+export default function TransactionItem({transaction}){
+ return(
+
+    <StyledTransactionItem key={transaction._id}>
```

Comment:
the key needs to be placed within the map, you can put it on the TransactionItem, you don't need to destructure it in the props.